Model/src/main/java/org/gusdb/wdk/model/query/param/FilterParamNewStableValue.java

```diff
@@ -512,6 +513,19 @@ String getValuesForSignature() {
 
     abstract List<?> getSortedMembers();
 
+    protected <T> String createInCondition(String metadataTableName, String columnName, List<T> membersList) {
```

Comment:
Is there any possibility that the strategy of multiple IN large in clauses will blow up the query planner?

Reply:
At least currently, this is an edge case (fungi has 1118 strains) so splitting one in clause into two is probably ok.  We could potentially see long running jobs in the future if for some reason we have many thousands of options in a membership filter, but we should be able to see that in the SQL and can address if it happens.  For now it is fine.